# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jbig2enc/sys-lepton.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- jbig2enc-0.27/Makefile.vanilla	2008-10-28 17:01:29.000000000 +0100
+++ jbig2enc-0.27/Makefile	2008-10-28 17:02:02.000000000 +0100
@@ -5,7 +5,7 @@
 CFLAGS=-I${LEPTONICA}/src -Wall -I/usr/include -L/usr/lib -O3 ${EXTRA}
 
 jbig2: libjbig2enc.a jbig2.cc
-	$(CC) -o jbig2 jbig2.cc -L. -ljbig2enc ${LEPTONICA}/src/liblept.a $(CFLAGS) -lpng -ljpeg -ltiff -lm
+	$(CC) -o jbig2 jbig2.cc -L. -ljbig2enc -llept $(CFLAGS) -lpng -ljpeg -ltiff -lm
 
 libjbig2enc.a: jbig2enc.o jbig2arith.o jbig2sym.o
 	ar -rcv libjbig2enc.a jbig2enc.o jbig2arith.o jbig2sym.o
@@ -18,7 +18,7 @@
 	$(CC) -c jbig2sym.cc -DUSE_EXT $(CFLAGS)
 
 delta: delta.c
-	$(CC) -o delta delta.c $(CFLAGS) ${LEPTONICA}/src/liblept.a -lpng -ljpeg -ltiff -lm
+	$(CC) -o delta delta.c $(CFLAGS) -llept -lpng -ljpeg -ltiff -lm
 
 clean:
 	rm -f *.o jbig2 libjbig2enc.a
